client/src/components/Modal/tests/Modal-test.js

```diff
@@ -0,0 +1,89 @@
+/* global jest, test, describe, it, expect */
+
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import Modal from '../Modal';
+
+let props;
+
+/** Component that printouts all of its props */
+const PrintProps = (componentName) => ({ children, ...propsToPrint }) => (
+  <div>
+    <div>{componentName}</div>
+    <ul>
+      {Object
+        .entries(propsToPrint)
+        .map(([key, value]) => (<li key={key}>{key}: {value.toString()}</li>))
+      }
+    </ul>
+    {children}
+  </div>
+);
```

Comment:
```suggestion
);
```
I don't think we need this, I put reasons why in corresponding deleted test where it was used